src/BytecodeLoader.cpp

```diff
@@ -117,44 +117,77 @@ game_value BytecodeLoader::buildCodeInstructions(const CompiledCodeData& data, c
         case InstructionType::push: {
             //#TODO if we push an array, we need to push a callUnary operator `+` to make sure it's copied.
             //Unless next instruction is something like params or forEach
+#ifdef ASC_INTERCEPT
             auto index = std::get<1>(it.content);
             instructions.emplace_back(GameInstructionConst::make(data.builtConstants[index])); //#TODO push of type array must be custom instruction that copies array on access
+#endif
         } break;
         case InstructionType::callUnary: {
+#ifdef ASC_INTERCEPT
             r_string name = std::get<0>(it.content);
+#else
+            r_string name = r_string(std::get<0>(it.content));
```

Comment:
non-intercept doesn't have r_string

Reply:
Intercept is still present in SQF-Assembly, but not in ASC